app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    if "/" in command_list[2]:
+        index_new_directories = command_list[2].rfind("/")
+        new_directories = command_list[2][:index_new_directories]
+        os.makedirs(new_directories, exist_ok=True)
+
+    if command_list[0] == "mv":
+        with (
+            open(command_list[1], "r") as file_riad,
+            open(command_list[2], "w") as file_write,
```

Comment:
The code does not handle exceptions that might occur during file operations, such as file not found or permission errors. Consider using a try-except block to handle potential exceptions.